app/controllers/carts_controller.rb

```diff
@@ -1,8 +1,28 @@
 class CartsController < ApplicationController
+
+  def index
+    @orders = Order.all
+  end
+
   def show
     @order_items = current_order.order_items
+    @products = Product.all
   end
   
-  def myorder
+  def myorder; end
+
+  def place_order
+    if current_order.sent == true
+      flash[:success] = "Order complete"
+      # current_order.save!
+      # session[:order_id] = nil   
```

Comment:
remove comment.

You can use the 1 line version here `flash[:success] = 'Order complete' if current_order.sent`